svy/src/helpers/balance.ts

```diff
@@ -21,4 +23,13 @@ export function sendSVY(accountAddress: Address, svySent: BigInt, block: ethereu
   account.lastUpdatedTimestamp = block.timestamp;
   account.save();
   return account;
+}
+
+export function updateVeSVYBalance(accountAddress: Address, block: ethereum.Block): Account {
+  const account = getOrCreateAccount(accountAddress);
+  account.veSVYBalance = veSVY.bind(Address.fromString(VE_SVY)).balanceOf(accountAddress);
```

Comment:
you can move `export veSVY = veSVY.bind(Address.fromString(VESVY_CONTRACT_ADDRESS))` to the Constants file